### PR TITLE
<hr> and <br>

### PR DESCRIPTION
**Pull request by pouvik**
updating all br / and hr / tags to br and hr to bring some uniformity.

signed off by me

### CODE REVIEW

**Comment by pouvik** [comment]
this is from convo from #6133 6133

File: Sources/Subs.php [hunk 1]
```diff
@@ -3291,7 +3291,7 @@ function parsesmileys(&$message)
 function highlight_php_code($code)
 {
 	// Remove special characters.
-	$code = un_htmlspecialchars(strtr($code, array('<br />' => "\n", '<br>' => "\n", "\t" => 'SMF_TAB();', '&#91;' => '[')));
+	$code = un_htmlspecialchars(strtr($code, array('<br>' => "\n", '<br>' => "\n", "\t" => 'SMF_TAB();', '&#91;' => '[')));
```
Comment (Gwenwyfar) [inline Sources/Subs.php]: Is this one right? Seems like it was correct as it was both `<br / >` and `<br>`

File: Themes/default/Xml.template.php [hunk 2]
```diff
@@ -107,7 +107,7 @@ function template_modifytopicdone()
 	}
 	else
 		echo '
-		<error in_subject="', $context['message']['error_in_subject'] ? '1' : '0', '"><![CDATA[', cleanXml(implode('<br />', $context['message']['errors'])), ']]></error>';
+		<error in_subject="', $context['message']['error_in_subject'] ? '1' : '0', '"><![CDATA[', cleanXml(implode('<br>', $context['message']['errors'])), ']]></error>';
```
Comment (Gwenwyfar) [inline Themes/default/Xml.template.php:110]: This is an implode, so maybe it was also correct?

File: Sources/Subs.php (hunk 1 above)
Comment (pouvik) [reply Sources/Subs.php]: possibly is, I just done a search :) ill fix now

File: Themes/default/Xml.template.php (hunk 2 above)
Comment (pouvik) [reply Themes/default/Xml.template.php:110]: this one I would think is fine as it outputing a break before error list (could be wrong just how I read the code)

**Review by Sesquipedalian — CHANGES_REQUESTED** [review]
Sorry, but most of these are mistaken changes that would cause bugs. The only correct ones are:

- the change to the img tag on line 1455 of Subs.php
- the changes to the 'cust_skype' field in the installer and upgrader files
- the changes to readme.html

Please revert all other changes.

**Comment by Sesquipedalian** [comment]
Unfortunately, your most recent commit to this pull request deleted a file that should not have been deleted. Because things have gotten a bit messy here, I am going to close this pull request and ask you to submit a new one that implements only the changes that were approved in this one.

**Pull request closed without merging**